cognite/client/_api/raw.py

```diff
@@ -358,7 +360,7 @@ def __call__(
         Returns:
             Iterator[Row] | Iterator[RowList]: An iterator yielding the requested row or rows.
         """
-        if partitions is None:
+        if partitions is None or _RUNNING_IN_BROWSER:
```

Comment:
Is this the main bugfix? Skip concurrency if pyodide

Reply:
Yup! Correct!